apps/server/src/auth/auth.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@nestjs/common';
+import { EmployeesService } from '../employees/employees.service';
+import { JwtService } from '@nestjs/jwt';
+import * as bcrypt from 'bcrypt';
+import { EmployeeEntity } from '../employees/entities/employee.entity';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private employeesService: EmployeesService,
+    private jwtService: JwtService,
+  ) {}
+
+  /**
+   * Validate an employee's credentials.
+   * @param email the employee's email
+   * @param pass the employee's raw text password
+   * @returns an employee
+   */
+  async validateEmployee(
+    email: string,
+    pass: string,
+  ): Promise<EmployeeEntity | null> {
+    const user = await this.employeesService.findOneByEmail(email);
+    if (user?.pswdHash && !(await bcrypt.compare(pass, user.pswdHash!))) {
+      return null;
+    }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Is this for eslint? Do we need to keep this line?

Reply:
we're basically splitting out the password from the object and using js destructuring to do it, but that means we have an unused variable that eslint complains abt. just telling eslint to ignore this one line for that error